starknet_py/serialization/serialization_test.py

```diff
@@ -110,9 +110,23 @@ class Education(NamedTuple):
 )
 parsed_abi = AbiParserV0(abi).parse()
 
-abi_v1 = json.loads(
-    load_contract(contract_name="ERC20", version=ContractVersion.V1)["sierra"]
-)["abi"]
+abi_v1 = [
+    {
+        "type": "event",
+        "name": "Approval",
+        "inputs": [
+            {
+                "name": "owner",
+                "type": "core::starknet::contract_address::ContractAddress",
+            },
+            {
+                "name": "spender",
+                "type": "core::starknet::contract_address::ContractAddress",
+            },
+            {"name": "value", "type": "core::integer::u256"},
+        ],
+    }
+]
```

Comment:
This contract has been removed because it has an event implemented inside and contracts written in Cairo V1 are not further supported by devnet if there are events. So this contract was only used for this test. I've deleted the code of this contract because it actually can't be declared and deployed on Devnet.

Reply:
Ok, maybe we can load it from separate file instead of having it here in code?

Reply:
I was inspired by this [file](https://github.com/software-mansion/starknet.py/blob/development/starknet_py/tests/e2e/fixtures/abi_v1_structures.py), so I can move it there or load it from a different file, what do you think?

Reply:
Ok I see, let's move it to `abi_v1_structures.py` then 👍

Reply:
Ok, let's move it to `abi_v1_structures.py` then 👍